### DISCUSSION TITLE
Can not see unsigned int datatype

### DISCUSSION THREAD

**Discussion by keshu-g** [supabase/supabase#28831] [Questions]
I need to store very big numbers in database from 0 to 40 Billion

the best option for me is 64 bit unsigned int 

but when I tried to find it in the docs I can not see anything that goes more then 8 bit

am i missing something ?

**Comment by GaryAustin1** [comment]
These are the Postgres datatypes: https://www.postgresql.org/docs/current/datatype-numeric.html
Not sure where you are seeing 8 bit?  Postgres does not have unsigned, but the 8byte int goes way higher than 40billion.
